app/structure/player.js

```diff
@@ -1,6 +1,5 @@
 export default class Player {
-    constructor(index, name, playerAlgorithm, snake) {
-        this.index = index;
+    constructor(name, playerAlgorithm, snake) {
```

Comment:
Index data is used in application please don't remove it.

Reply:
See https://github.com/FrostF0X/snake_wars/pull/1#discussion_r157685229 :D